### ISSUE TITLE
Default AIMNet2 units

### ISSUE THREAD

**Issue by MasterLucas** [isayevlab/aimnetcentral#88]
Dear Sir/ Madam, 

I've been trying to fine-tune the aimnet2-wb97m-d3 model, but the results I got on my test set seem quite off. What are the default force and energy units AIMNet2 uses? This was the configuration used in my `train-config.yaml` file for the forces and energy:
```
energy;
  abbr: E
  peratom: True
  scale: 1

forces:
  abbr: F
  peratom: True
  mult: 3
  scale: 1
```

Best regards,

Lucas Bandeira

**Comment by isayev** [comment]
Default units are eV for energies and eV/A for forces — the published models' atomic-energy shifts (SAE) are in eV, and the calculator returns eV / eV/A. For fine-tuning, your dataset's energies and forces must be in those units (with `scale: 1`); a dataset in Hartree or kcal/mol without conversion would produce exactly the kind of systematic offset you're seeing. Closing as answered — if results still look off with eV-unit data, please share the training curves in a new issue.
